python/ray/function_manager.py

```diff
@@ -230,14 +230,14 @@ def get_execution_info(self, job_id, function_descriptor):
         Returns:
             A FunctionExecutionInfo object.
         """
-        if self._worker.load_code_from_local:
+        try:
             # Load function from local code.
             # Currently, we don't support isolating code by jobs,
             # thus always set job ID to NIL here.
             job_id = ray.JobID.nil()
             if not function_descriptor.is_actor_method():
                 self._load_function_from_local(job_id, function_descriptor)
-        else:
+        except RuntimeError as e:
```

Comment:
What's the performance impact for users who want to load code from GCS (which is the majority)?

Reply:
Let's remove it in another pr